feature-libs/my-account/package.json

```diff
@@ -9,18 +9,34 @@
     "storefront",
     "my account"
   ],
+  "scripts": {
+    "clean:schematics": "yarn && rimraf \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
```

Comment:
```suggestion
    "clean:schematics": "rimraf \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
```

Reply:
Running yarn on every test run is a bit of overkill ;)

Reply:
It's required to install the `rimraf` dep. I can try to run `yarn` in the ci script